pkg/cmd/scan/output/testdata/output.json

```diff
@@ -76,5 +76,6 @@
 		}
 	],
 	"coverage": 33,
-	"alerts": null
+	"alerts": null,
+	"ProviderVersion": ""
```

Comment:
I'm wondering if it's normal that the ProviderVersion string is empty ?

Reply:
I think he forgot to add it in the existing output test suites